test/http.test.js

```diff
@@ -1,22 +0,0 @@
-/* eslint-disable no-unused-expressions */
-/* eslint-disable no-undef */
-import * as Factory from '../lib/factory.js';
-chai.use(chaiAsPromised);
-const expect = chai.expect;
-
-describe('http manager', () => {
```

Comment:
I don't see this test re-added in the new dir.

Reply:
This is an old test file, and it's not even loaded in https://github.com/standardnotes/snjs/blob/master/test/test.html